LICENSES/MIT.txt

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) <year> <copyright holders>
```

Comment:
Placeholder? Is this correct?

Reply:
Yeah, reuse recommends to keep the license text unmodified like that: https://reuse.software/faq/#license-templates